tensorflow/core/kernels/mkl/mkl_eltwise_activation_base_op.h

```diff
@@ -0,0 +1,288 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Please add a newline before and after this line. Same for other new files.

Reply:
Done